java/src/main/java/vowpalWabbit/learner/VWLearners.java

```diff
@@ -104,4 +104,6 @@ private static void loadNativeLibrary() {
 
     // Closing needs to be done here when initialization fails and by VWBase
     static native void closeInstance(long nativePointer);
+
+    static native void saveModel(long nativePointer, String filename);
```

Comment:
This needs to be callable from `VWBase`, so it can't be private. `closeInstance` above is the same. They are package-private though, so users of the library can't call them directly.